test/ampycloud/ref_data/Geneva_2018.04.30-18.17.00_SCT100-BKN120.csv

```diff
@@ -0,0 +1,274 @@
+,ceilo,dt,alt,type
```

Comment:
The first column with the indexes is not required. Removing it will remove a warning when running the tests.

Reply:
Done, thanks!